.github/workflows/preview.yaml

```
@@ -41,24 +43,13 @@ jobs:
         run: |
           if (test -a _build/html/reports/*log); then cat _build/html/reports/*log ; fi
 
-      - name: Publish to Surge
-        id: preview_step
-        uses: afc163/surge-preview@v1
-        with:
-          surge_token: ${{ secrets.SURGE_TOKEN }}
-          github_token: ${{ secrets.GITHUB_TOKEN }}
-          dist: _build/html
-          build: |
-            npm install surge
+      - name: Install Surge.sh
+        uses: actions/setup-node@v4
 
-      - name: Format preview url
-        id: format_url
-        run: |
-          echo "url_text=:rocket: Preview deployed at https://${{ steps.preview_step.outputs.preview_url }}" >> $GITHUB_OUTPUT
+      - run: npm install -g surge
 
-      - name: Get the preview_url
-        run: echo "${{ steps.format_url.outputs.url_text }}"
+      - run: surge ./_build/html {{env.PREVIEW_URL}} --token ${{ secrets.SURGE_TOKEN }}
```

```suggestion
      - run: surge ./_build/html {{env.PREVIEW_URL}} --token $SURGE_TOKEN
        env:
          SURGE_TOKEN: ${{ secrets.SURGE_TOKEN }}
```
Set surge token to env so it can be properly accessed per https://docs.github.com/en/actions/security-guides/using-secrets-in-github-actions#example-using-bash